packages/lit-html/src/parts.ts

```diff
@@ -76,35 +76,43 @@ export const setPartValue = <T extends Part>(
         "An index must be provided to set an AttributePart's value."
       );
     }
-    const newValues = [...(part._value as Array<unknown>)];
+    const newValues = [...(part._$value as Array<unknown>)];
     newValues[index] = value;
-    (part as AttributePart)._setValue(newValues, 0);
+    (part as AttributePart)._$setValue(newValues, 0);
   } else {
-    part._setValue(value);
+    part._$setValue(value);
   }
   return part;
 };
 
-export const getPartValue = (part: NodePart) => part._value;
+export const getPartValue = (part: NodePart) => part._$value;
+
+// A sentinal value that can never appear as a part value except when set by
+// live(). Used to force a dirty-check to fail and cause a re-render.
+const RESET_VALUE = {};
+
+export const resetPartValue = (part: Part, value: unknown = RESET_VALUE) =>
```

Comment:
Why is this here? Add to PR description?

Reply:
Added to PR description:
> * Adds `resetPartValue` helper for use by directives that were accessing the private `_value` property